plugins/threatminer/help.md

```diff
@@ -901,21 +897,22 @@ An example raw response from the domain action:
 ```
 
 Working through the `[]results` object array which resides in the `result` object, will require some JSON manipulation to get what you need.
-The [jq](https://market.komand.com/plugins/komand/jq/0.1.0) and [JSON](https://market.komand.com/plugins/komand/json/0.1.1) plugins are great at sifting through the data.
-
-#
+The [jq](https://extensions.rapid7.com/extension/jq) and [JSON](https://extensions.rapid7.com/extension/json-edit) plugins are great at sifting through the data.
 
+## Troubleshooting
+  
+*There is no troubleshooting for this plugin.*
 
 # Version History
 
+* 3.0.0 - Updated SDK to the latest version | Updated packages | Added unittests
```

Comment:
Based on the log, should this be a patch update or has something else changed?

Reply:
Oh, removed one action as it's not handled by the API anymore. I will extend the changelog 😄